src/renderer/assets/images/icons/index.tsx

```diff
@@ -1,27 +1,201 @@
-const settings = require('./Settings.svg');
-const play = require('./Play.svg');
-const bell = require('./Bell.svg');
-const add = require('./Add.svg');
-const preferences = require('./Preferences.svg');
-const popup = require('./Popup.svg');
+import { ReactComponent as Add } from './Add.svg';
```

Comment:
Thanks for adding all these! Will be nice to have icons typed

Reply:
just want to make sure, but when you say that it'll "be nice to have icons typed", did I miss something regarding the icon types? or was this about using `ReactComponent as __`?

Reply:
I think it will be helpful being able to use IconId from `export type IconId = keyof Icons;`!